synapse/tests/test_lib_base.py

```diff
@@ -241,25 +264,26 @@ def wootfunc(mesg):
         await bref.fini()
         self.true(base0.isfini)
 
-    @s_glob.synchelp
+        await base2.fini()
+
     async def test_base_waitfini(self):
+        loop = s_glob.plex.loop
 
         base = await s_base.Base.anit()
 
         self.false(await base.waitfini(timeout=0.1))
 
         async def callfini():
-            await asyncio.sleep(0.1, loop=asyncio.get_event_loop())
+            await asyncio.sleep(0.1, loop=loop)
             await base.fini()
 
-        asyncio.get_event_loop().create_task(callfini())
+        loop.create_task(callfini())
```

Comment:
this test should also confirm that base.isinfi is true

Reply:
ok